library/src/main/java/com/bumptech/glide/load/engine/bitmap_recycle/BitmapCopier.java

```diff
@@ -0,0 +1,34 @@
+
+package com.bumptech.glide.load.engine.bitmap_recycle;
+import android.graphics.Bitmap;
+import android.graphics.Matrix;
+import androidx.annotation.Nullable;
+/**
+ * Copies {@link android.graphics.Bitmap}s based on desired configurations.
+ *
+ * <p>Unlike {@link BitmapPool}, the methods should preserve additional {@link Bitmap} metadata such
+ * as gainmaps and color spaces.
+ */
+public final class BitmapCopier {
+  private BitmapCopier() {}
+
+  /**
+   * Creates a bitmap that is identical to the input, preserving properties such as color space
+   * and gainmap, allowing for an optional transformation to be specified by a {@link Matrix}.
+   *
+   * @param inBitmap the source bitmap to copied and transformed
+   * @param matrix the transformation matrix to apply to the pixels of {@code inBitmap} and any
+   *     associated gainmap pixels if non-null.
+   *
+   */
+  public static Bitmap copy(Bitmap inBitmap, @Nullable Matrix matrix) {
```

Comment:
I was addressing this comment: https://github.com/bumptech/glide/pull/5334#discussion_r1417857479. Quoting here: 
"I'm totally on board with ditching the BitmapPooll. But you're going to need to write this in more places (at least all other Transformations in this class) to have it work consistently, especially for local images.

Might be worth thinking through earlier rather than later how we'd go about it. A static helper? A BitmapHelper interface that is like the BitmapPool but only ever provides new instances?"

The idea is that we would use this class to replace other usages of BitmapPool. Of course, I could just remove the boilerplate as you suggest, but I would need to put all the javadoc somewhere.

Reply:
If we're planning on removing the bitmap pool entirely you don't need this documentation at all, you'd just remove it and replace it with Bitmap.createBitmap calls right?

Reply:
I removed the boilerplate.